geo-data-server/src/test/java/org/gridsuite/geodata/test/EmbeddedCassandraFactoryConfig.java

```diff
@@ -42,7 +42,7 @@ Cassandra cassandra(CassandraFactory cassandraFactory) {
     @Bean
     CassandraConnection cassandraConnection(Cassandra cassandra) {
         CassandraConnection cassandraConnection = new DefaultCassandraConnectionFactory().create(cassandra);
-        CqlDataSet.ofClasspaths("create_keyspace.cql", "geo_data.cql").forEachStatement(cassandraConnection::execute);
+        CqlDataSet.ofClasspaths("create_keyspace.cql").add(CqlDataSet.ofStrings("USE geo_data;")).add(CqlDataSet.ofClasspaths("geo_data.cql")).forEachStatement(cassandraConnection::execute);
```

Comment:
It may be better to use the keyspace parameter in order to have a complete test
See comment for `network-store-server`

Reply:
Done